paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -98,66 +116,109 @@ def get_reviewers(filename):
     return authors[:3]
 
 
-def review(filename):
+def review(filename, description, over):
     reviewers = ' '.join(get_reviewers(filename))
-    description = (
-        'This change was made automatically by the perf_update_soaconfigs script. If not reviewed'
-        ' in a week, it will be merged by the perf or paasta teams. For more context on why these'
-        ' changes are being made see: PERF-2439'
-    )
     subprocess.check_call((
-        'review-branch',
+        'git',
+        'push',
         '--force',
-        '--summary=automatically updating {} for under/overprovisioned mem/cpu'.format(filename),
+        'origin',
+        'HEAD',
+    ))
+    subprocess.check_call((
+        'review-branch',
+        '--summary=automatically updating {} for {}provisioned cpu'.format(filename, over),
         '--description="{}"'.format(description),
         '--reviewers', reviewers,
-        '--target-groups', 'operations',
+        '--server', 'https://reviewboard.yelpcorp.com',
+        '--target-groups', 'operations perf',
     ))
 
 
-def edit_soa_configs(filename, instance, mem, cpu):
+def edit_soa_configs(filename, instance, cpu):
     with open(filename, 'r') as fi:
         yams = fi.read()
         yams = yams.replace('cpus: .', 'cpus: 0.')
         data = yaml.round_trip_load(yams, preserve_quotes=True)
 
     instdict = data[instance]
-    if mem:
-        instdict['mem'] = mem
-    else:
-        instdict['cpus'] = cpu
+    instdict['cpus'] = cpu
     out = yaml.round_trip_dump(data, width=10000)
 
     with open(filename, 'w') as fi:
         fi.write(out)
 
 
+def create_jira_ticket(serv, creds, description):
+    creds = creds.split(':')
+    options = {'server': 'https://jira.yelpcorp.com'}
+    jira_cli = JIRA(options=options, basic_auth=(creds[0], creds[1]))
+    jira_ticket = {}
+    # Sometimes the project name doesn't match a JIRA project
+    try:
+        jira_ticket = {
```

Comment:
Can you put this on line 156, and then only in the exception case just change the project name and try again?